java/src/jmri/jmrix/cmri/CMRISystemConnectionMemo.java

```diff
@@ -12,6 +13,8 @@
 
     public CMRISystemConnectionMemo() {
         super("C", CMRIConnectionTypeList.CMRI);
+        this.register();
```

Comment:
In NetBeans at least, I can rely on the code completion from this... to ensure I get the method signature correct and complete.